site/guide/work-with-content-blocks.qmd

```diff
@@ -41,11 +41,18 @@ Test-driven block
 6. Click {{< fa square-plus >}} and then select one of the available options: 
     
     - **Simple text block**: Adds a new section with a blank content block. After the new content block has been added, click {{< fa pencil >}} to edit the contents of the section like any other.
-    - **Test-driven block**: Select one of the options:
-        - **Metric**: Select one of the available metrics, such as **Confusion Matrix**.
-        - **Threshold test**: Select one of the available threshold tests, such as **Data Quality: Skewness** or **Model Diagnosis: Overfit Regions**.
-    
-7. For test-driven blocks, a preview of the available metrics or threshold test gets shown. Click **Insert module** when you are ready.
+    - **Test-driven block**: Adds a new section with test results. 
+
+7. For test-driven blocks, a menu containing available tests will appear:
+
+
+      ![](test-driven-block-menu.png){width=85%}
+
+   - Select tests using the checkboxes on the left.
+   - Search by name using **{{<fa magnifying-glass >}} Search** on the top-left.
+   - Click on a test to preview it. By default, selected tests are previewed.
+
+8. Click **Insert test result to document** when you are ready.
```

Comment:
```suggestion
8. Click **Insert _n_ test result to document** when you are ready.
```

It can be hard to match buttons that change based on what you select but this gets you a bit closer.